docs/tutorials/github.md

```diff
@@ -0,0 +1,93 @@
+# GitHub Tutorial
+
+This tutorial provides an introduction to using `trestlebot` with GitHub.  We will be using a single GitHub repository for our trestle authoring workspace and executing the `trestlebot` commands as GitHub actions.  Note, each repo is intended to support authoring a single OSCAL model type (SSP, component definition, etc.).  If authoring more than one, then a dedeicated repository should be used for each model.
+
+
+### 1. Prerequisites
+
+Before moving on, please ensure you have completed the following:
+
+1. create a new (or use an existing) empty GitHub repository
+2. clone the repo to your local workstation
+3. clone the [trestle-bot](https://github.com/RedHatProductSecurity/trestle-bot/tree/main) repo to your local workstation and run `poetry install`
+
+
+### 2. Set Permissions for GitHub Actions
+
+The `trestlebot` commands will be run inside of GitHub actions.  These commands often perform `write` level operations against the repo contents.  The following repo settings need to be in place to support these actions.
+
+1. Click the `Settings` tab for your GitHub repo 
+2. Select `Actions` -> `General` from the left-hand menu
+3. Scroll down to `Workflow permissions`
+4. Ensure `Read repository contents and packages permissions` is selected
+5. Ensure `Allow GitHub Actions to create and approve pull requests` is checked
```

Comment:
It might be worth adding a note that this is only if a user wants to use the `GITHUB_TOKEN` in GitHub Actions and they could also use a GitHub App or PAT.